### ISSUE TITLE
Build bot: request a fresh build NOW

### ISSUE THREAD

**Issue by Korkman** [Korkman/macemu-appimage-builder#10]
Comments in any issue within this repository with any number of the following key phrases will trigger a new automatic build:

- Hey build bot, please build BasiliskII-x86_64.AppImage!
- Hey build bot, please build BasiliskII-i386.AppImage!
- Hey build bot, please build BasiliskII-aarch64.AppImage!
- Hey build bot, please build SheepShaver-x86_64.AppImage!
- Hey build bot, please build SheepShaver-i386.AppImage!
- Hey build bot, please build SheepShaver-aarch64.AppImage!

The bot will immediately react with a rocket emoji, then send a reply when the new build is ready to download on the releases page. Multiple phrases can be posted in one comment to trigger multiple builds at once. Edits are not parsed. The bot will answer with a success message only if all builds succeed. Otherwise, a failure message will be posted (unless [the bot](https://github.com/Korkman/macemu-appimage-builder/actions/workflows/Build%20bot.yml) is broken).

Build times are typically below 5 minutes. Please do not waste resources. Check the modification times of the individual files on the [continuous release page](https://github.com/Korkman/macemu-appimage-builder/releases/tag/continuous).

**Comment by Korkman** [comment]
Hey build bot, please build macemu-i386-basilisk2.AppImage!

**Comment by Korkman** [comment]
Hey build bot, please build macemu-i386-basilisk2.AppImage!

**Comment by Korkman** [comment]
Hey build bot, please build macemu-amd64-basilisk2.AppImage!

**Comment by Korkman** [comment]
Hey build bot, please build macemu-amd64-sheepshaver.AppImage!

**Comment by Korkman** [comment]
Hey build bot, please build macemu-i386-sheepshaver.AppImage!
Hey build bot, please build macemu-amd64-basilisk2.AppImage!

**Comment by Korkman** [comment]
Something something not build bot

**Comment by Korkman** [comment]
Hey build bot, please build macemu-i386-sheepshaver.AppImage!
Hey build bot, please build macemu-amd64-basilisk2.AppImage!

**Comment by Korkman** [comment]
Something something not build bot

**Comment by Korkman** [comment]
Hey build bot, please build macemu-i386-sheepshaver.AppImage!

**Comment by Korkman** [comment]
Hey build bot, please build macemu-amd64-sheepshaver.AppImage!
Hey build bot, please build macemu-i386-sheepshaver.AppImage!

**Comment by Korkman** [comment]
Hey build bot, please build macemu-amd64-sheepshaver.AppImage!
Hey build bot, please build macemu-i386-sheepshaver.AppImage!

**Comment by Korkman** [comment]
Hey build bot, please build nothing!

**Comment by Korkman** [comment]
Hey build bot, please build nothing!

**Comment by Korkman** [comment]
Hey build bot, please build a skyscraper!

**Comment by Korkman** [comment]
Hey build bot, please build nothing!

**Comment by Korkman** [comment]
Hey build bot, please build a skyscraper!

**Comment by Korkman** [comment]
Hey build bot, please build nothing!

**Comment by Korkman** [comment]
Hey build bot, please build a skyscraper!

**Comment by Korkman** [comment]
Hey build bot, please build nothing!

**Comment by Korkman** [comment]
Hey build bot, please build a skyscraper!

**Comment by Korkman** [comment]
Hey build bot, please build a skyscraper!

**Comment by Korkman** [comment]
Hey build bot, please build nothing!

**Comment by Korkman** [comment]
Hey build bot, please build macemu-amd64-basilisk2.AppImage!
    Hey build bot, please build macemu-i386-basilisk2.AppImage!
    Hey build bot, please build macemu-amd64-sheepshaver.AppImage!
    Hey build bot, please build macemu-i386-sheepshaver.AppImage!

**Comment by Korkman** [comment]
@kanjitalk755 You might find this build bot useful to push important bugfixes to the community, if such a need ever arises. Thank you for your work on macemu, highly appreciated.

**Comment by Korkman** [comment]
Periodic refresh of the continuous build is set to every Friday.

**Comment by cat7** [comment]
Hey build bot, please build macemu-amd64-sheepshaver.AppImage!

**Comment by Korkman** [comment]
Refreshing the build to include latest (and final) vhd library version. Still only supports the original VHD format, not VHDX.
    Hey build bot, please build macemu-amd64-basilisk2.AppImage!
    Hey build bot, please build macemu-i386-basilisk2.AppImage!
    Hey build bot, please build macemu-amd64-sheepshaver.AppImage!
    Hey build bot, please build macemu-i386-sheepshaver.AppImage!

**Comment by Korkman** [comment]
Hey build bot, please build macemu-amd64-basilisk2.AppImage!
    Hey build bot, please build macemu-i386-basilisk2.AppImage!
    Hey build bot, please build macemu-amd64-sheepshaver.AppImage!
    Hey build bot, please build macemu-i386-sheepshaver.AppImage!

**Comment by Korkman** [comment]
New naming scheme

    Hey build bot, please build BasiliskII-x86_64.AppImage!
    Hey build bot, please build BasiliskII-i386.AppImage!
    Hey build bot, please build SheepShaver-x86_64.AppImage!
    Hey build bot, please build SheepShaver-i386.AppImage!

**Comment by Korkman** [comment]
Hey build bot, please build BasiliskII-x86_64.AppImage!
    Hey build bot, please build BasiliskII-i386.AppImage!
    Hey build bot, please build SheepShaver-x86_64.AppImage!
    Hey build bot, please build SheepShaver-i386.AppImage!

**Comment by Korkman** [comment]
Hey build bot, please build BasiliskII-x86_64.AppImage!
Hey build bot, please build BasiliskII-i386.AppImage!
Hey build bot, please build SheepShaver-x86_64.AppImage!
Hey build bot, please build SheepShaver-i386.AppImage!

**Comment by Korkman** [comment]
Hey build bot, please build BasiliskII-x86_64.AppImage!
    Hey build bot, please build BasiliskII-i386.AppImage!
    Hey build bot, please build SheepShaver-x86_64.AppImage!
    Hey build bot, please build SheepShaver-i386.AppImage!

**Comment by Korkman** [comment]
```
Hey build bot, please build BasiliskII-x86_64.AppImage!
Hey build bot, please build BasiliskII-i386.AppImage!
Hey build bot, please build SheepShaver-x86_64.AppImage!
Hey build bot, please build SheepShaver-i386.AppImage!
```

**Comment by Korkman** [comment]
@Spz2022 Yes, it works that way. But the files were renamed long ago. See either top or last comment for examples. Took the liberty of deleting all comments with old filenames.

**Comment by Spz2022** [comment]
Hey build bot, please build SheepShaver-x86_64.AppImage!

**Comment by Korkman** [comment]
Hey build bot, please build BasiliskII-x86_64.AppImage!
    Hey build bot, please build BasiliskII-i386.AppImage!
    Hey build bot, please build SheepShaver-x86_64.AppImage!
    Hey build bot, please build SheepShaver-i386.AppImage!

**Comment by Korkman** [comment]
Hey build bot, please build SheepShaver-x86_64.AppImage!

**Comment by Korkman** [comment]
Hey build bot, please build BasiliskII-x86_64.AppImage!
Hey build bot, please build BasiliskII-i386.AppImage!
Hey build bot, please build BasiliskII-aarch64.AppImage!
Hey build bot, please build SheepShaver-x86_64.AppImage!
Hey build bot, please build SheepShaver-i386.AppImage!
Hey build bot, please build SheepShaver-aarch64.AppImage!

**Comment by Korkman** [comment]
Hey build bot, please build BasiliskII-x86_64.AppImage!
Hey build bot, please build BasiliskII-i386.AppImage!
Hey build bot, please build BasiliskII-aarch64.AppImage!
Hey build bot, please build SheepShaver-x86_64.AppImage!
Hey build bot, please build SheepShaver-i386.AppImage!
Hey build bot, please build SheepShaver-aarch64.AppImage!

**Comment by Korkman** [comment]
Hey build bot, please build BasiliskII-x86_64.AppImage!
Hey build bot, please build BasiliskII-i386.AppImage!
Hey build bot, please build BasiliskII-aarch64.AppImage!
Hey build bot, please build SheepShaver-x86_64.AppImage!
Hey build bot, please build SheepShaver-i386.AppImage!
Hey build bot, please build SheepShaver-aarch64.AppImage!
